geomstats/geometry/riemannian_metric.py

```diff
@@ -460,7 +461,7 @@ def while_loop_body(iteration, mean, variance, sq_dist):
             maximum_iterations=n_max_iterations)
 
         if last_iteration == n_max_iterations:
-            print('Maximum number of iterations {} reached.'
+            warnings.warn('Maximum number of iterations {} reached.'
```

Comment:
The continuous integration tests fail because of this line (flake8 complains). Could you pep8 the style? Thanks!

Reply:
See: https://travis-ci.org/geomstats/geomstats/builds/642679593?utm_source=github_status&utm_medium=notification

Reply:
forgot to run flake8 one last time before submitting PR. Just fixed it